platform/wab/src/wab/shared/folders/folders-util.ts

```diff
@@ -62,6 +66,9 @@ function insertIntoTree<T>(
   }
 
   const newPath = `${path}${index !== 0 ? "-" : ""}${folders[index]}`;
+  const newNormalPath = `${normalPath}${index !== 0 ? "/" : ""}${
```

Comment:
I needed the current path of the folder, and figured it's better to track it here than rebuild from the code that uses it.

I could move the functionality to a helper and calculate the path on the fly, but I think it ends up being much more complex.

Reply:
Maybe in the past it was used in the DOM? As far as I can tell, it's only currently used as a render key.

Reply:
Oh, I realized it's not so easy. Slashes are fine, but the path should not have a prefix, since it would be treated as a group/folder. For example, clicking add on `Pages/Group1` would prompt the user to add something like `Pages/Group1/Cmp1`, which would be placed in the hierarchy as `Pages/Pages/Group1/Cmp1`.

An alternative is to use a dash only for the prefix, and extract later (e.g. `Pages-Group1/Group2`), or adjust the types so that the prefix is guaranteed (and strip later), but I'm not sure if that's more maintainable.

Reply:
I think we can use the `getFolderWithSlash` to handle the removal of the prefix, if we use `Pages-Group1/Group2` and it would be clean. What do you think?

Reply:
Hmm, now that I think of it, underscore could be used in the component name itself, so we need to use a separator that is not used in javascript identifiers, like `~`?

Reply:
The updated code should be searching from the start, so it shouldn't be a problem. I can switch to `~` if you think it's clearer though, and test to see if I can break it.